### PR TITLE
Import shipitapi

### LINKED ISSUE #46: Integrate shipitapi
One of the requirements of shipitscript is shipitapi, which isn't used anywhere else AFAIK. We should integrate it into shipitscript in order to reduce deps and archive the original repo.

### PR DESCRIPTION
**Pull request by rail**
Closes #46

* switch tp pip-compile-multi
* blackify

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -14,16 +14,16 @@ passenv =
...
-    pytest
-    pytest-asyncio
-    pytest-cov
+    -rrequirements/test.txt
 
 commands=
     py.test --cov-config .coveragerc --cov=shipitscript --cov-report term-missing
     coverage html
     flake8
+    pip-compile-multi verify
```
Comment (MihaiTabara) [inline tox.ini]: Nice additional checks!

File: requirements/base.in [hunk 2]
```diff
@@ -0,0 +1,5 @@
+certifi
```
[the remark this answers is not in the document]
Comment (rail) [reply requirements/base.in]: I'm going to remove them. Good catch.

File: requirements/test.in [hunk 3]
```diff
@@ -0,0 +1,13 @@
+-r base.in
+
+black
+check-manifest
+coverage
+flake8
+isort
+mock
+pip-compile-multi
```
[the remark this answers is not in the document]
Comment (rail) [reply requirements/test.in]: Yeah, there is `pip-compile-multi verify` in `commands`

File: tox.ini (hunk 1 above)
Comment (rail) [reply tox.ini]: \o/

File: shipitscript/script.py [hunk 4]
```diff
@@ -8,11 +8,11 @@
 
 from shipitscript import ship_actions
 from shipitscript.task import (
```
[the remark this answers is not in the document]
Comment (rail) [reply shipitscript/script.py]: \o/

**Pull request merged**